### PR TITLE
Move chancellor code over.

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**

### CODE REVIEW

**Comment by josephbridgwaterrowe** [comment]
@michaelachrisco please review.

**Comment by michaelachrisco** [comment]
Ignoring the hound errors?

**Comment by josephbridgwaterrowe** [comment]
I think its talking bum bum as I updated the configuration to ignore vendor files.

**Comment by michaelachrisco** [comment]
All files are passing with Rubocop so I am inclined to agree.

File: node_modules/bower/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - '0.10'
+  - '0.11'
+matrix:
+  allow_failures:
+    - node_js: '0.11'
+script:
+  - grunt travis
```
Comment (michaelachrisco) [inline node_modules/bower/.travis.yml]: Interesting. Ill have to look into https://www.npmjs.com/package/grunt-travis

**Comment by michaelachrisco** [comment]
I can't find too much given that we have already looked at most of the code. I am inclined to merge except for the issue we were discussing earlier with mysql. Ill try this on postgres and give you an update.

**Comment by michaelachrisco** [comment]
I think it was the brew install of postgresql. Merging.

**Pull request merged**